### DISCUSSION TITLE
which is the lasted? 1.9.0.45 or 1.9.0.1105

### DISCUSSION THREAD

**Discussion by gwnpeter** [nanoframework/Home#1449] [Q&A]
nanoff cannot download 1.9.0.45. but i found that 1.9.0.45 maybe the lasted version ....

**Comment by gwnpeter** [comment]
![1710168989550](https://github.com/nanoframework/Home/assets/162097756/f1e9b06c-28ef-4230-89d4-9ac2e5f03ae6)

**Comment by alberk8** [comment]
I guess the numbering got haywired.  To get the version you want add --fwversion 1.9.0.45 to the end of nanoff command.

**Comment by gwnpeter** [comment]
thx
